doc/whats_new/v1.0.rst

```diff
@@ -1118,6 +1118,9 @@ Changelog
   `n_features_in_` and will be removed in 1.2. :pr:`20240` by
   :user:`Jérémie du Boisberranger <jeremiedbb>`.
 
+- |Fix| :meth:`preprocessing.FunctionTransformer._check_inverse_transform`
```

Comment:
You need to move this in 1.1 and check the warning raised by the documentation CI